scripts/figures/single/map_twocenters.R

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8-unix -*-
```

Comment:
Everything in this file is done four times in a row. Could transform this into a function that is called four times?

Reply:
Fixed in https://github.com/HSLdevcom/mal-effect-calculations/pull/30/commits/fa6225c174c8fc3ba779b41f2e06a2e075187ea2 https://github.com/HSLdevcom/mal-effect-calculations/pull/30/commits/328c40b67dcf4d52a94c2a2b7a3b8413550d1f1e https://github.com/HSLdevcom/mal-effect-calculations/pull/30/commits/2d3a56c30375252ed7521db2a257dd048183b65c